lib/torrent.js

```diff
@@ -509,6 +505,22 @@ Torrent.prototype._onMetadata = function (metadata) {
     return new File(self, file)
   })
 
+  // Only select specified files (BEP53)
+  if (self.so) {
+    var specificFiles = parseRange.parse(self.so)
+    // https://github.com/webtorrent/webtorrent/issues/164
+    self.deselect(0, self.pieces.length - 1, false)
```

Comment:
I believe that this line should not be necessary since you moved the `self.select(0, self.pieces.length - 1, false)` call to an `else` block below. So, what is this actually deselecting?

Reply:
Initially I had the `if (self.so)` statement under `self.select` all without the `else` statement, this is leftover code from that which can be removed, removed in next commit.